src/SprykerEco/Zed/PunchoutCatalogs/Communication/Form/PunchoutCatalogConnectionForm.php

```diff
@@ -29,12 +31,15 @@ class PunchoutCatalogConnectionForm extends AbstractType
 {
     public const OPTION_BUSINESS_UNIT_CHOICES = 'OPTION_BUSINESS_UNIT_CHOICES';
     public const OPTION_CONNECTION_FORMAT_SUB_FORM_TYPES = 'OPTION_CONNECTION_FORMAT_FORMS';
+    public const OPTION_CONNECTION_TYPE_SUB_FORM_TYPES = 'OPTION_CONNECTION_TYPE_SUB_FORM_TYPES';
 
     protected const BUSINESS_UNIT_FIELD_PLACEHOLDER = 'Choose business unit';
     protected const BUSINESS_UNIT_FIELD_LABEL = 'Business unit';
 
     protected const VALIDATION_GROUP_DISABLED = 'disabled';
 
+    protected const DEPENDENT_GROUP_TYPE = 'type';
```

Comment:
`ATTRIBUTE_DEPENDENT_GROUP_TYPE` ?

Reply:
Renamed to `ATTRIBUTE_VALUE_DEPENDENT_GROUP_TYPE `, since it's not an attribute itself.